pkg/cmd/roachtest/run.go

```diff
@@ -209,6 +227,80 @@ func runTests(register func(registry.Registry), filter *registry.TestFilter) err
 	return err
 }
 
+// predictTestsToRun filters the tests to run based on certain criteria:
+// 1. the number of time a test has been successfully running
+// 2. the test is new
+// 3. the test has not been run for a while
+// 4. a subset of the successful tests
+func predictTestsToRun(
```

Comment:
This function name is not very good as there is no "prediction" going on. In fact you are selecting the tests that will be run.

Reply:
will change it to selectTestsToRun.